inspirehep/config.py

```diff
@@ -877,7 +878,7 @@ def _(x):
 
 # Web services and APIs
 # =====================
-BEARD_API_URL = None  # e.g. "http://beard.inspirehep.net/api"
+BEARD_API_URL = "http://beard.inspirehep.net/api"
```

Comment:
This and the following, would be better kept disabled. But I guess it's needed for testing, until #1282 is implemented.